### PR TITLE
add Obis GetLorawanInfo command

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: docs/obis-observer/commands/GetLorawanInfo.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### Examples
+
+#### LoRaWAN information:
+
+| Field                 | Value | Hex    |
+| --------------------- | ----- | ------ |
+| command id            | `31   | `0x1f` |
+| request id            | `7`   | `0x07` |
+| device EUI            | `[0x00, 0x1a, 0x79, 0x88, 0x16, 0xaa, 0x55, 0x61]` | |
+| application EUI       | `[0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77]` | |
```
Comment (DarkPark) [inline docs/obis-observer/commands/GetLorawanInfo.md:49]: the last column should have value

something like this:

```suggestion
| application EUI       | `[0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77]` | `0x0011223344556677` |
```

File: docs/obis-observer/commands/GetLorawanInfo.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| Connection method `1` | `1`   | `0x01` |
```
Comment (DarkPark) [inline docs/obis-observer/commands/GetLorawanInfo.md:50]: is the first column correct here?

File: docs/obis-observer/commands/GetLorawanInfo.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+| `8`  | `byte`                               | application EUI                                 |
+| `1`  | [Connection method](../types.md#LoRaWAN-connection-method) | LoRaWAN connection method |
+
+### Examples
+
+#### LoRaWAN information:
+
+| Field                 | Value | Hex    |
+| --------------------- | ----- | ------ |
+| command id            | `31   | `0x1f` |
```
[the remark this answers is not in the document]
Comment (Pnom) [reply docs/obis-observer/commands/GetLorawanInfo.md]: implemented

File: docs/obis-observer/commands/GetLorawanInfo.md (hunk 1 above)
Comment (Pnom) [reply docs/obis-observer/commands/GetLorawanInfo.md]: implemented

File: docs/obis-observer/commands/GetLorawanInfo.md (hunk 2 above)
Comment (Pnom) [reply docs/obis-observer/commands/GetLorawanInfo.md]: don't understand

Comment (Pnom) [reply docs/obis-observer/commands/GetLorawanInfo.md]: implemented

**Review by DarkPark — APPROVED** [review]

**Pull request merged**